lib/Backends/CPU/Transforms.cpp

```diff
@@ -121,6 +121,9 @@ static Node *optimizeCPUMaxSplat(MaxNode *MN, Function *F) {
 
 bool CPUBackend::transformPostLowering(Function *F,
                                        CompilationContext &) const {
+  // Log the start of current log scope.
```

Comment:
Nit: I'm not sure that this comment that you use everywhere in this PR adds a lot of value.

Reply:
LOG_SCOPE seems not to be directly relevant with the what is going on inside each optimization function, so i thought it might be better to add a comment here for developers to read

Reply:
My point was that the SCOPE declaration code is so self-describing that the comment does not add much. But it is really a very minor comment.

Reply:
I agree, the comment isn't adding anything that isn't in the macro name.

Reply:
sounds reasonable, i'll just remove them